contracts/ReaperAutoCompoundScreamLeverage.sol

```diff
@@ -370,13 +365,10 @@ contract ReaperAutoCompoundScreamLeverage is ReaperBaseStrategy {
             blockShareBorrow = borrowed * distributionPerBlock / totalBorrow;
         }
 
-        //how much we expect to earn per block
+        // How much we expect to earn per block
         uint256 blockShare = blockShareSupply + blockShareBorrow;
         uint256 secondsPerBlock = 1; // Average FTM block speed
-
-        //last time we ran harvest
-        uint256 lastHarvestTime = harvestLog[harvestLog.length - 1].timestamp;
```

Comment:
using the last log entry (instead of `lastHarvestTimestamp`) might have contributed to reporting inflated rewards. It's possible that a harvest happened 45 minutes ago but wasn't logged because the log cadence is 1 hour. In this case our rewards would only accrue over 45 minutes.

Reply:
It would make sense if this happened to TUSD